ui/ui-expenses/src/screens/prices/capture/capture.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Webcam from 'react-webcam'
+import { component } from 'ui-hocs'
+
+const Capture = ({
+  device,
+  height,
+  width,
+  capture,
+  useCamera,
+  takeCapture,
+  cancel,
+}) => {
+  if (capture) {
+    return (
+      <div>
+        <img width={50} height={50} src={capture} />
+        <button onClick={cancel}>Cancel</button>
+      </div>
+    )
+  }
+  return (
+    <div>
+      <Webcam
+        audio={false}
+        height={height}
+        width={width}
+        ref={useCamera}
+        screenshotFormat="image/webp"
```

Comment:
Can you save to jpeg, so we can process the image like we do with screenshot etc

Reply:
ok, just to see the size of picture with this format